### DISCUSSION TITLE
Making my own div.

### DISCUSSION THREAD

**Discussion by sasja-san** [quarto-dev/quarto-cli#9049] [Q&A]
### Description

I want to be able to create a `tcolorbox` latex output with quarto. Either extending the `callout` notation or creating my own.

My goal is to be able to create something like the green box here. This is from page 184 in the tcolorbox manual.
![tcolorbox-green-with-fancy-title](https://github.com/quarto-dev/quarto-cli/assets/712688/c4fcbf94-d2e3-4211-8d62-49eeefb390f7)

As I have no idea on how the source code is laid out it seems better to ask before wasting hours upon hours in the wrong place.

Should I modify the source of quarto? Or is there some way to have my box generating code reside inside of the project where I aim to use it? Is this documented and I just missed it?

**Comment by mcanouil** [comment]
I believe you are looking for the LaTeX Environment extension: https://github.com/quarto-ext/latex-environment

**Reply by sasja-san** [reply]
Thanks. I would never have found that on my own.

**Reply by mcanouil** [reply]
It's listed in the listing of extensions: https://quarto.org/docs/extensions/

(and https://m.canouil.dev/quarto-extensions/extensions.html)
